### PR TITLE
hub-api: add org management api specs

### PR DESCRIPTION
**Pull request by ivan-californias**
## Description

Add Org management endpoints openapi specs and fix formating issues.

## Related issues or tickets

https://docker.atlassian.net/browse/IAM-842

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/reference/api/hub/latest.yaml [hunk 1]
```diff
@@ -6,112 +6,104 @@ info:
...
+    x-audience: public
+    url: https://hub.docker.com
 tags:
   - name: resources
     x-displayName: Resources
     description: |
       The following resources are available to interact with the documented API:
-
-      - <a href="https://github.com/docker/hub-tool#readme">Docker Hub CLI tool</a> (currently experimental)
+      - [Docker Hub CLI tool]("https://github.com/docker/hub-tool#readme) (currently experimental)
```
Comment (sarahsanders-docker) [inline content/reference/api/hub/latest.yaml:25]: this goes to a 404

File: content/reference/api/hub/latest.yaml [hunk 2]
```diff
@@ -6,112 +6,104 @@ info:
...
-      scopes are assumed. For example: If you define `repo:write`, the API assumes the scope of both 
-      `repo:read` *and* `repo:public_read` as well. If you were to define both `repo:write` *and* 
-      `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
+      For each scope grouping (in this case "repo"), you only need to define 1 scope as any lower scopes are assumed.
+      For example: If you define `repo:write`, the API assumes the scope of both `repo:read` *and* `repo:public_read` as well.
+      If you were to define both `repo:write` *and* `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
 
-      ***Treat your personal access token like your password and keep it secret. You cannot retrieve 
-      your token after it is generated.***
+      ***Treat your personal access token like your password and keep it secret. You cannot retrieve your token after it is generated.***
```
Comment (sarahsanders-docker) [inline content/reference/api/hub/latest.yaml:64]:
```suggestion
      **Warning**: Treat your personal access token like your password and keep it secret. You cannot retrieve your token after it is generated.
```

File: content/reference/api/hub/latest.yaml [hunk 3]
```diff
@@ -6,112 +6,104 @@ info:
...
+      The groups endpoints allow you to manage your organization's teams and their members.
+
+      For more information, seee [Create and manage a team](https://docs.docker.com/admin/organization/manage-a-team/).
+  - name: invites
+    x-displayName: Invites
+    x-audience: public
+    description: |
+      The invites endpoints allow you to manage invites for users to join your Docker organization.
+
+      For more information, see [Invite members](https://docs.docker.com/admin/organization/members/#invite-members).
```
Comment (sarahsanders-docker) [inline content/reference/api/hub/latest.yaml:99]: I am not able to see any of these in the preview: https://deploy-preview-21615--docsdocker.netlify.app/reference/api/hub/latest/#tag/repositories

File: content/reference/api/hub/latest.yaml [hunk 4]
```diff
@@ -552,202 +531,722 @@ paths:
...
-          $ref: "#/components/responses/NotFound"
-
-
+        '200':
+          description: Repository tag exists
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+  /v2/orgs/{org_name}/members:
```
Comment (sarahsanders-docker) [inline content/reference/api/hub/latest.yaml:620]: again, not able to see any of the org members stuff in the preview: https://deploy-preview-21615--docsdocker.netlify.app/reference/api/hub/latest/

File: content/reference/api/hub/latest.yaml (hunk 1 above)
Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml:25]: Doesn't for me 🤔

File: content/reference/api/hub/latest.yaml (hunk 2 above)
Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml:64]: AFAIK this is auto rendering and we shouldn't change it. Can check though

File: content/reference/api/hub/latest.yaml (hunk 1 above)
Comment (sarahsanders-docker) [reply content/reference/api/hub/latest.yaml:25]: weird. if I copy and paste the URL it is fine, but 404s when I click the link in the docs preview

Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml:25]: Hmm can dig a bit further here

File: content/reference/api/hub/latest.yaml (hunk 3 above)
Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml:99]: Fixed

File: content/reference/api/hub/latest.yaml (hunk 4 above)
Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml:620]: Fixed

File: content/reference/api/hub/latest.yaml (hunk 2 above)
Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml:64]: I think we really want bold and italic to stand out here.

File: content/reference/api/hub/latest.yaml (hunk 1 above)
Comment (technicallyjosh) [reply content/reference/api/hub/latest.yaml]: Fixed

**Review by sarahsanders-docker — DISMISSED** [review]

**Review by sarahsanders-docker — DISMISSED** [review]

**Review by sarahsanders-docker — APPROVED** [review]

**Pull request merged**